### DISCUSSION TITLE
Binary Executable Size

### DISCUSSION THREAD

**Discussion by catalinux17** [pocketbase/pocketbase#251] [Q&A] [answered]
I see that pocketbase release v0.3.2 is 11mb worth of size, when I try to go build it with `-ldflags "-s -w"` I got it down to 34mb. All I've added are some migrations (but that doesn't make a difference from what I've tested).
Any idea how can I get it smaller?

**Comment by ganigeorgiev** [comment, marked as answer]
~11MB is the compressed zip archive size. When extracted the binary is around ~35MB.

If you want to "shrink" your binary size you could use something like upx (https://upx.github.io/).
Based on my local tests in the past, this could reduce the size to under 10MB. But keep in mind that packing the binary may also increase slightly the startup time of your application because it is decompressed on the fly (that is the main reason why is not prepacked by default).
